lib/cog/queries/bundles.ex

```diff
@@ -27,4 +28,11 @@ defmodule Cog.Queries.Bundles do
     select: b.id,
     limit: 1
   end
+
+  def exists?(id) do
+    Repo.one!(from r in Bundle,
+              where: r.id == ^id,
+              select: count(r.id)) == 1
+  end
```

Comment:
How is this being used? Curious what the use case would be where access to the bundle model itself isn't also needed.